index.js

```diff
@@ -33,7 +33,12 @@ module.exports = {
     if(opts.temp) {
       var name = isString(opts.temp) ? opts.temp : ''+Date.now()
       opts.path = path.join(osenv.tmpdir(), name)
-      rimraf.sync(opts.path)
+
+      if (!opts.startUclean) {
```

Comment:
I think there may be a typo here. :wink: 

Also, if possible I think it would be nice to specify that we're looking for `true` specifically.

```suggestion
      if (opts.startUnclean !== true) {
```